pylint/checkers/spelling.py

```diff
@@ -308,9 +333,25 @@ def _check_spelling(self, msgid, line, line_num):
             initial_space = 0
         if line.strip().startswith("#") and "docstring" not in msgid:
             line = line.strip()[1:]
+            # A ``Filter`` cannot determine if the directive is at the beginning of a line, nor determine if a colon is present or not (``pyenchant`` strips trailing colons). So implementing this here.
+            for iter_directive in (
```

Comment:
This could be an option in the pylintrc, what do you think ?

Reply:
Oh, I see.  it's just the `.config`.  Yep, I think I can do that.  I'll have it default to this pre-existing list if none is present in pylintrc if that's OK.